docs/ACME.md

```diff
@@ -41,8 +41,8 @@ placed in Synapse's config directory without the need for any ACME setup.
 
 The main steps for enabling ACME support in short summary are:
 
-1. Allow Synapse to listen on port 80 with authbind, or forward it from a reverse-proxy.
-1. Set `acme:enabled` to `true` in homeserver.yaml.
+1. Allow Synapse to listen on port 80 with `authbind`, or forward it to port `8009` from a reverse-proxy.
+1. Set `acme:enabled` to `true` and `acme:port` to the appropriate port (`8009` for reverse-proxy, `80` for listening directly with `authbind`) in homeserver.yaml.
```

Comment:
you don't need to set acme:port if its port 80, and it's starting to just be a duplicate of the information below. Can we make this snappier and handwavier: "enable acme support in homeserver.yaml".